test/e2e/v1alpha1/e2e_image_mutate_test.go

```diff
@@ -62,28 +90,152 @@ var _ = Describe("For a Kubernetes cluster with Tekton and build installed", fun
 			testBuild.ValidateImageDigest(buildRun)
 
 			image := testBuild.GetImage(buildRun)
-
-			Expect(
-				getImageAnnotation(image, "org.opencontainers.image.url"),
-			).To(Equal("https://my-company.com/images"))
-
-			Expect(
-				getImageLabel(image, "maintainer"),
-			).To(Equal("team@my-company.com"))
+			Expect(annotationsOf(image)).To(HaveKeyWithValue("org.opencontainers.image.url", "https://my-company.com/images"))
+			Expect(labelsOf(image)).To(HaveKeyWithValue("maintainer", "team@my-company.com"))
```

Comment:
Unrelated to timestamp work, but file needed to touched anyway and this should serve to have a common style and to be more readable.